reView/cli.py

```diff
@@ -83,3 +83,49 @@ def unpack_turbines(
         out_gpkg_path.unlink(missing_ok=True)
 
     turbines_gdf.to_file(out_gpkg_path, driver='GPKG')
+
+
+@main.command()
+@click.option('--supply_curve_csv', '-i', required=True,
+              prompt='Path to input bespoke wind supply curve CSV file',
+              type=click.Path(exists=True, dir_okay=False, file_okay=True),
+              help='Path to bespoke wind supply curve CSV file created by reV')
+@click.option('--char_map', '-m', required=True,
+              prompt='Path to JSON file storing characterization map',
+              type=click.Path(exists=True, dir_okay=False, file_okay=True),
+              help='Path to JSON file storing characterization map')
+@click.option('--out_csv', '-o', required=True,
+              prompt='Path to output csv.',
+              type=click.Path(dir_okay=False),
+              help='Path to CSV to store results')
+@click.option('--cell_size', '-c', required=False,
+              default=90.,
+              type=float,
+              help=('(Optional) Cell size in meters of characterization '
```

Comment:
Doesn't `click` tell the use automatically if a parameter is optional? Also you don't list the following parameter as `(Optional)`, which is slightly inconsistent.